platform/src/main/resources/db/migration/V6__add_bank_account_type.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE sep31_transaction ADD bank_account_type VARCHAR(255);
```

Comment:
It'll collide with versioning on #745 
If we merge this PR first it's a good idea for @stfung77 to rename his migration file to V7